pkg/controllers/workapplier/controller_integration_test.go

```diff
@@ -3952,3 +3952,615 @@ var _ = Describe("report diff", func() {
+		})
+
+		It("should prepare an AppliedWork object", func() {
+			appliedWorkCreatedActual := appliedWorkCreatedActual(workName)
+			Eventually(appliedWorkCreatedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to prepare an AppliedWork object")
+
+			appliedWorkOwnerRef = prepareAppliedWorkOwnerRef(workName)
+		})
+
+		It("should own the objects, but not apply any manifests", func() {
+			// Verify that the Deployment manifest has not been applied, yet Fleet has assumed
+			// its ownership.
+			wantDeploy := deploy.DeepCopy()
+			wantDeploy.TypeMeta = metav1.TypeMeta{}
+			wantDeploy.Namespace = nsName
+			wantDeploy.Name = deployName
+			wantDeploy.OwnerReferences = []metav1.OwnerReference{
+				*appliedWorkOwnerRef,
+			}
+			wantDeploy.Spec.Replicas = ptr.To(int32(2))
+
+			deployOwnedButNotApplied := func() error {
+				if err := memberClient.Get(ctx, client.ObjectKey{Namespace: nsName, Name: deployName}, regularDeploy); err != nil {
+					return fmt.Errorf("failed to retrieve the Deployment object: %w", err)
+				}
+
+				if len(regularDeploy.Spec.Template.Spec.Containers) != 1 {
+					return fmt.Errorf("number of containers in the Deployment object, got %d, want %d", len(regularDeploy.Spec.Template.Spec.Containers), 1)
+				}
+				if len(regularDeploy.Spec.Template.Spec.Containers[0].Ports) != 1 {
+					return fmt.Errorf("number of ports in the first container, got %d, want %d", len(regularDeploy.Spec.Template.Spec.Containers[0].Ports), 1)
+				}
+
+				// To ignore default values automatically, here the test suite rebuilds the objects.
+				rebuiltGotDeploy := &appsv1.Deployment{
+					ObjectMeta: metav1.ObjectMeta{
+						Namespace:       regularDeploy.Namespace,
+						Name:            regularDeploy.Name,
+						OwnerReferences: regularDeploy.OwnerReferences,
+					},
+					Spec: appsv1.DeploymentSpec{
+						Replicas: regularDeploy.Spec.Replicas,
+						Selector: regularDeploy.Spec.Selector,
+						Template: corev1.PodTemplateSpec{
+							ObjectMeta: metav1.ObjectMeta{
+								Labels: regularDeploy.Spec.Template.ObjectMeta.Labels,
+							},
+							Spec: corev1.PodSpec{
+								Containers: []corev1.Container{
+									{
+										Name:  regularDeploy.Spec.Template.Spec.Containers[0].Name,
+										Image: regularDeploy.Spec.Template.Spec.Containers[0].Image,
+										Ports: []corev1.ContainerPort{
+											{
+												ContainerPort: regularDeploy.Spec.Template.Spec.Containers[0].Ports[0].ContainerPort,
+											},
+										},
+									},
+								},
+							},
+						},
+					},
+				}
+
+				if diff := cmp.Diff(rebuiltGotDeploy, wantDeploy); diff != "" {
+					return fmt.Errorf("deployment diff (-got +want):\n%s", diff)
+				}
+				return nil
+			}
+
+			Eventually(deployOwnedButNotApplied, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to own the Deployment object without applying the manifest")
```

Comment:
ca we reduce the eventuallyDuration to be 10 or even 5 second?

Reply:
It should work, let me update the value.